public/js/components/App.js

```diff
@@ -41,7 +41,7 @@ const App = React.createClass({
   },
 
   toggleSourcesSearch(e) {
-    if (e.metaKey && e.key == "p") {
+    if (e.metaKey && e.key == "p" || e.ctrlKey && e.key == "p") {
```

Comment:
Thanks! It's a little shorter to do `if ((e.metaKey || e.ctrlKey) && e.key === "p") ...`, do you mind changing it to that?